src/main/java/com/jakewharton/rxrelay/Relay.java

```diff
@@ -26,6 +26,22 @@ protected Relay(OnSubscribe<R> onSubscribe) {
   }
 
   /**
+   * Portrays a object of an Action1 implementation as a simple Relay object. This is useful, for instance,
```

Comment:
Hmm, yeah you're right. Was trying to keep the language consistent but it's definitely backward here. I'll fix it